File: Asp.Net Mvc/BlogMvc.Web/Models/Comment.cs
namespace BlogMvc.Web.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    [Table("comments")]
    public partial class Comment
    {
        [Key]
        public int Id { get; set; }
        public int Post_Id { get; set; }

        [Required()]
        public string Username { get; set; }

        [Required()]
        [EmailAddress()]
        public string Mail { get; set; }


        [Required()]
        public string Content { get; set; }
        public System.DateTime Created { get; set; }


        [ForeignKey("Post_Id")]
        public virtual Post Post { get; set; }
    }
}

File: Yii2/models/Comment.php
<?php

namespace app\models;

use Yii;

/**
 * This is the model class for table "comments".
 *
 * @property integer $id
 * @property integer $post_id
 * @property string $username
 * @property string $mail
 * @property string $content
 * @property string $created
 *
 * @property Posts $post
 */
class Comment extends \yii\db\ActiveRecord
{
    /**
     * @inheritdoc
     */
    public static function tableName()
    {
        return 'comments';
    }

    /**
     * @inheritdoc
     */
    public function rules()
    {
        return [
            [['post_id', 'username', 'mail', 'content', 'created'], 'required'],
            [['post_id'], 'integer'],
            [['content'], 'string'],
            [['created'], 'safe'],
            [['username', 'mail'], 'string', 'max' => 255]
        ];
    }

    /**
     * @inheritdoc
     */
    public function attributeLabels()
    {
        return [
            'id' => 'ID',
            'post_id' => 'Post ID',
            'username' => 'Username',
            'mail' => 'Mail',
            'content' => 'Content',
            'created' => 'Created',
        ];
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getPost()
    {
        return $this->hasOne(Post::className(), ['id' => 'post_id']);
    }
}
